packages/core/src/render3/instructions/instructions.ts

```diff
@@ -1142,6 +1169,34 @@ export function elementAttribute(
 }
 
 /**
+ * Update a property on a selected element.
+ *
+ * Operates on the element selected by index via the {@link select} instruction.
+ *
+ * If the property name also exists as an input property on one of the element's directives,
+ * the component property will be set instead of the element property. This check must
+ * be conducted at runtime so child components that add new `@Inputs` don't have to be re-compiled
+ *
+ * @param propName Name of property. Because it is going to DOM, this is not subject to
+ *        renaming as part of minification.
+ * @param value New value to write.
+ * @param sanitizer An optional function used to sanitize the value.
+ * @param nativeOnly Whether or not we should only set native properties and skip input check
+ * (this is necessary for host property bindings)
+ * @returns This function returns itself so that it may be chained
+ * (e.g. `property('name', ctx.name)('title', ctx.title)`)
+ */
+export function property<T>(
+    propName: string, value: T, sanitizer?: SanitizerFn | null, nativeOnly?: boolean):
+    any {  // NOTE: `any` returned here because tsickle freaked out over recursive return
```

Comment:
Let's make this a TODO with the relevant issue linked

```suggestion
    any {  // TODO: Update from `any` when tsickle bug with recursive returns is fixed (https://github.com/angular/tsickle/issues/1009)
```

Reply:
You're too quick! :) Done... but moved up to the type that @mhevery suggested.